CHANGELOG.md

```diff
@@ -18,7 +27,7 @@ _Note: Gaps between patch versions are faulty, broken or test releases._
 ## 2.9.0
  * **New Feature**
    * Show chunk sizes in sidebar (closes #91)
-   
```

Comment:
My editor removes trailing spaces automatically. I can readd them if wanted.

Reply:
It's fine :+1: